### ISSUE TITLE
How does the blocking work?

### ISSUE THREAD

**Issue by mymebenha** [MicrosoftDocs/microsoft-365-docs#7571]
I've tried the block list and the messages that were going to quarantine are still going to quarantine.  If by "block" you mean block from the inbox and send to quarantine then the article should mention that.  I've had to use a mail rule instead to actually block the message. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 94b60171-271c-fb93-88bf-0cd9db014541
* Version Independent ID: e9cd207e-0d4c-e268-dd3a-701e1e2df658
* Content: [Manage your allows and blocks in the Tenant Allow/Block List - Office 365](https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/tenant-allow-block-list?view=o365-worldwide)
* Content Source: [microsoft-365/security/office-365-security/tenant-allow-block-list.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/tenant-allow-block-list.md)
* Product: **m365-security**
* Technology: **mdo**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@mymebenha, yes. Block or Allow means "interrupt/override the results of the protection stack and block or allow delivery of the message to the intended recipient". The results of the block could be "deliver to quarantine" or "deliver to the Junk Email folder".

So what are you trying to do?

**Comment by mymebenha** [comment]
Thanks for the info, it wasn't clear from the article.  Several users have the same email going to their quarantine everyday and we want to completely block it so they don't need to review it everyday.  Doesn't sound like there's an option here so I'll try using exchange mail rules instead.  Thanks

**Comment by chrisda** [comment]
@mymebenha, ah, got it. You want an Allow. Do you know why the messages are being quarantined? That will determine the best way to do the allow. If the block is due to spoofing, you can use the Tenant Allow/Block List and/or the [Spoof intelligence insight](https://docs.microsoft.com/en-/microsoft-365/security/office-365-security/learn-about-spoof-intelligence). If the block is due to user/domain (sender) impersonation, then you can use the [Impersonation insight](https://docs.microsoft.com/microsoft-365/security/office-365-security/impersonation-insight).

Other than that, yeah, mail flow rules (transport rules) are the best way to do the allow. Just be sure to make the rules as narrow or specific as you can.

**Comment by chrisda** [comment]
@mymebenha, OK, so you do want a Block. Anti-spam policies allow you to silently delete messages as the action for all spam filtering verdicts except high confidence phishing. But you would need to be very careful about scoping the policy to specific users and specific messages so you don't throw out good messages with bad. As you say, transport rules might work best.

But honestly, what's wrong with quarantining bad messages? They'll age out after 30 days, anyway. Deleting messages is a one-way ticket that you can't ever recover from. At least quarantine provides a safety net in case a good message is misidentified.

[Quarantine policies](https://docs.microsoft.com/microsoft-365/security/office-365-security/quarantine-policies) allow you to say what users are able to do to those quarantined messages.

**Comment by mymebenha** [comment]
Normally quarantining bad messages is what we want but in this particular case a spammy email is going to several users in the organization almost daily so I was looking for a way to stop it from appearing in their quarantine \ junk on a daily basis.  If there were a way to only show the message in quarantine online for admins, that would be sufficient.  They've tried the block button on the quarantine report but that didn't' seem to do anything.  The transport rule is working though.

**Comment by chrisda** [comment]
@learningdaily, you should open a support ticket and report the undesirable behavior.
